### DISCUSSION TITLE
HELP.airflow datahub integration. Authentication chain failed to resolve a valid authentication

### DISCUSSION THREAD

**Discussion by wwwvikas123** [apache/airflow#26210] [Q&A]
The problem with airflow datahub integration. Datahub set as linage backend for Airflow. DAG works, but it shows an error sometimes. Thanks in advance.
METADATA_SERVICE_AUTH_ENABLED is true. We use token to access GMS.
Airflow log
![image](https://user-images.githubusercontent.com/85458456/188881337-ffcf3fc5-12b1-4cc8-94c3-0bcd4b7b5523.png)
Datahub log
![image](https://user-images.githubusercontent.com/85458456/188881455-7c5bc77c-470c-4a75-a749-8336ca130a4e.png)

Connection in Airflow
![image](https://user-images.githubusercontent.com/85458456/188881621-73b2d569-cabd-4be4-8f19-4ce151c45c33.png)

**Comment by potiuk** [comment]
You should as at datahub - this is not a community managed provider and it's managed by Datahub I guess https://github.com/datahub-project/datahub -is the right place to ask.
